ktlint-ruleset-standard/src/test/resources/spec/unary-op-spacing/format.kt.spec

```diff
@@ -0,0 +1,17 @@
+fun main() {
```

Comment:
Could you also add a tests for `val g = 3 + i` or `val k = i - 3` or another similar "normal" math cases?

Reply:
of course. "math case" added in this commit

https://github.com/pinterest/ktlint/pull/798/commits/f719fb9e84a22d12570c920f814a88fdef59182c